[2022-01-29 14:07:29,558] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:29,661] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:29,661] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:29,661] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-29 14:07:29,662] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:29,819] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2019-04-02 06:00:00+00:00
[2022-01-29 14:07:29,831] {standard_task_runner.py:52} INFO - Started process 313 to run task
[2022-01-29 14:07:29,993] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2019-04-02T06:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmpydqw3tt_', '--error-file', '/tmp/tmpjq_5_g1z']
[2022-01-29 14:07:29,994] {standard_task_runner.py:77} INFO - Job 21: Subtask get_data
[2022-01-29 14:07:30,597] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-04-02T06:00:00+00:00 [running]> on host d1e1fe005577
[2022-01-29 14:07:31,079] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T06:00:00+00:00
[2022-01-29 14:07:31,082] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 14:07:31,083] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "helloworld!!"']
[2022-01-29 14:07:31,100] {subprocess.py:85} INFO - Output:
[2022-01-29 14:07:31,106] {subprocess.py:89} INFO - helloworld!!
[2022-01-29 14:07:31,107] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 14:07:31,182] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20190402T060000, start_date=20220129T140729, end_date=20220129T140731
[2022-01-29 14:07:31,248] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 14:07:31,329] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
